### DISCUSSION TITLE
GTK: ability to load custom CSS

### DISCUSSION THREAD

**Discussion by jcollie** [ghostty-org/ghostty#4088] [Feature Requests, Ideas] [answered]
Add a config entry `gtk-custom-css` that, when set, will load custom CSS from a file. The setting can be repeated to load multiple files and files can be marked as optional by prefixing it with a `?`. The custom CSS will be added as priority USER so it should take precedence over any CSS set by Ghostty or the system

**Comment by mitchellh** [comment, marked as answer]
#4089 Approved
